src/pages/api/thirdparty/schema.ts

```diff
@@ -0,0 +1,22 @@
+import { gql } from 'graphql-tag'
+
+/* Data we will return to any third-party querying the API. */
```

Comment:
It's a bit repetitive to define these typeDefs when these are already defined in the Keystone CMS GraphQL API. We could potentially import the entire schema from Keystone and use that for typeDefs, but then we'd have to manually update it every time a change was made in Keystone, and also would be limited to returning only the data that Keystone exposes.

Reply:
Thought: I think it's fine to have a little repetition. While the typeDefs are similar now they may not be in the future and the audience of each is different.